zoo/example.txt

```diff
@@ -0,0 +1,24 @@
+Basic usage:
+
+Set up of a simple zoo with two cages for testing.
+
+from zoo.zoo import Zoo, Cage, BaseSpecies, Lion, Hyena, Wildebeest, Gazelle
```

Comment:
`zoo.zoo` seems verbose.  See if you can work out how to make it work like this:
    from zoo import Zoo